examples/03_connectivity/plot_fast_clustering.py

```diff
@@ -0,0 +1,149 @@
+
+start = time.time()
+rena = ReNA(scaling=True, n_clusters=2000, mask=nifti_masker,
+            memory='nilearn_cache')
+
+rena.fit_transform(func_filename)
+print("ReNA 2000 clusters: %.2fs" % (time.time() - start))
+
+##################################################################
+# Visualize results
+# ------------------
+#
+# First we display the labels of the clustering in the brain.
+#
+# To visualize results, we need to transform the clustering's labels back
+# to a neuroimaging volume. For this, we use the NiftiMasker's
+# inverse_transform method.
+from nilearn.plotting import plot_roi, plot_epi, show
+import numpy as np
+
+# Avoid 0 label
+labels = rena.labels_ + 1
+# Shuffling the labels for visualization
+permutation = np.random.permutation(labels.shape[0])
+labels = permutation[labels]
+# Unmask the labels
+labels_img = nifti_masker.inverse_transform(labels)
+
+##################################################################
+# labels_img is a Nifti1Image object, it can be saved to file with the
+# following code:
+labels_img.to_filename('parcellation.nii')
+
+
+from nilearn.image import mean_img
+mean_func_img = mean_img(func_filename)
+
+first_plot = plot_roi(labels_img, mean_func_img, title="ReNA parcellation",
+                      display_mode='xz')
+
+# common cut coordinates for all plots
+cut_coords = first_plot.cut_coords
+
+##################################################################
+# Second, we illustrate the effect that the clustering has on the
+# signal. We show the original data, and the approximation provided by
+# the clustering by averaging the signal on each parcel.
+#
+# As you can see below, this approximation is very good, although there
+# are only 2000 parcels, instead of the original 60000 voxels
+
+# The transform call extracts the time-series from the files:
+fmri_masked = nifti_masker.transform(func_filename)
+
+# Display the original data
+plot_epi(nifti_masker.inverse_transform(fmri_masked[0]),
+         title='Original (%i voxels)' % fmri_masked.shape[1],
+         cut_coords=cut_coords, vmax=fmri_masked.max(), vmin=fmri_masked.min(),
+         display_mode='xz')
+
+
+# A reduced data can be created by taking the parcel-level average:
+# Note that, as many scikit-learn objects, the ReNA object exposes
+# a transform method that modifies input features. Here it reduces their
+# dimension.
+# However, the data are in one single large 4D image, we need to use
+# index_img to do the split easily:
+from nilearn.image import index_img
+fmri_reduced_rena = rena.transform(index_img(func_filename, 0))
+
+# Display the corresponding data compression using the parcellation
+compressed_img_rena = rena.inverse_transform(fmri_reduced_rena)
+
+plot_epi(compressed_img_rena, cut_coords=cut_coords,
+         title='ReNA: Compressed representation (2000 parcels)',
+         vmax=fmri_masked.max(), vmin=fmri_masked.min(),
+         display_mode='xz')
+
+show()
```

Comment:
The two examples look somewhat redundant.

Reply:
Yeah as above, I forgot to remove the one specific to ReNA